.github/workflows/build.yml

```diff
@@ -43,17 +43,20 @@ jobs:
           path: source/
 
   run_tests:
-    name: Run tests
+    name: Run tests (GHC ${{ matrix.ghc }})
     needs: package_source
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        ghc: ['8.0', '8.10', '9.0.1', 'latest']
```

Comment:
The oldest and newest 8.x and 9.x provided by https://github.com/haskell-actions/setup.